.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Unintentional? This folder should probably be in gitignore

Reply:
Yep. Good catch. Thx!